### PR TITLE
feat: Use the common schemas for more properties of various schemas

### PR DESCRIPTION
**Pull request by wolfy1339**

### CODE REVIEW

**Comment by wolfy1339** [comment]
@G-Rath It's complaining that there is no schema for an event, but there is.
Can you take a look?

File: payload-schemas/schemas/common/repository.schema.json [hunk 1]
```diff
@@ -429,6 +429,9 @@
         }
       },
       "additionalProperties": false
+    },
+    "public": {
+      "type": "boolean"
```
Comment (gr2m) [inline payload-schemas/schemas/common/repository.schema.json:434]: I'm pretty sure that the `"public"` key is always present, can you add it to the `"required"` array?

**Review by gr2m — COMMENTED** [review]
Just a small nit, otherwise looks good to me

File: payload-schemas/schemas/common/repository.schema.json (hunk 1 above)
Comment (wolfy1339) [reply payload-schemas/schemas/common/repository.schema.json:434]: The `"private"` key is always present, but not `"public"`.
I checked the payload for `repository.transferred`, `repository.created` and the `"public"` key is not included in the `"repository"` object.

It seems that it's only used for the `fork` event from what I can tell

Comment (gr2m) [reply payload-schemas/schemas/common/repository.schema.json:434]: You are right, sorry, I mixed up the two

**Review by gr2m — APPROVED** [review]

**Pull request merged**